helm/archive-node/templates/archive.yaml

```diff
@@ -17,11 +17,13 @@ spec:
         role: archive-node
         version: {{ trunc 6 (split ":" .Values.coda.image)._1 | trimSuffix "-" }}
       annotations:
-        {{- if .Values.archive.enableLocalDaemon }}
         prometheus.io/scrape: 'true'
+        {{- if .Values.archive.enableLocalDaemon }}
         prometheus.io/port: {{ .Values.coda.ports.metrics | quote }}
-        prometheus.io/path: '/metrics'
+        {{ else }}
+        prometheus.io/port: {{ .Values.archive.ports.metrics | quote }}
```

Comment:
I think for archive node pods, we should enable archive metrics whether or not it has a local daemon. If all the archive pods have a local daemon then we won't be getting any archive metrics. There are a lot more daemon metrics coming from other pods anyway, so it should be ok to skip the archive daemon metrics until we can have both the ports enabled.